src/components/CreateManifestModal/CreateManifestModal.tsx

```diff
@@ -1,48 +1,60 @@
-import React, { FC } from 'react';
-import { Modal, ModalVariant } from '@patternfly/react-core';
+import React, { FC, useEffect } from 'react';
+import { Modal } from '@patternfly/react-core/dist/dynamic/components/Modal';
+import { ModalVariant } from '@patternfly/react-core/dist/dynamic/components/Modal';
 import CreateManifestForm from '../CreateManifestForm/CreateManifestForm';
 import CreateManifestFormLoading from '../CreateManifestForm/CreateManifestFormLoading';
 import useCreateSatelliteManifest from '../../hooks/useCreateSatelliteManifest';
 import useSatelliteVersions from '../../hooks/useSatelliteVersions';
+import { supportLink } from '../../utilities/consts';
+import useNotifications from '../../hooks/useNotifications';
 
 interface CreateManifestModalProps {
   handleModalToggle: () => void;
   isModalOpen: boolean;
 }
-
 const CreateManifestModal: FC<CreateManifestModalProps> = ({ handleModalToggle, isModalOpen }) => {
   const {
     data,
     isLoading: isLoadingSatelliteVersions,
     isError: hasSatelliteVersionsError
   } = useSatelliteVersions();
-
+  const { addErrorNotification } = useNotifications();
   const {
     data: createManifestResponseData,
     mutate,
     isLoading: isCreatingManifest,
     isSuccess,
     isError: errorCreatingManifest,
-    reset: resetCreateSatelliteManifestQuery
+    reset: resetCreateSatelliteManifestQuery,
+    error: createManifestError
   } = useCreateSatelliteManifest();
 
   const submitForm = (name: string, version: string) => {
     mutate({ name, version });
   };
 
-  if (isModalOpen === false && isSuccess === true) {
+  if (isModalOpen && createManifestResponseData) {
     resetCreateSatelliteManifestQuery();
   }
 
-  /**
-   * In case of an error, the API will "succeed",
-   * but with an error status, so we need to
-   * check for data to confirm success
-   */
-
   const hasCreatedManifest = typeof createManifestResponseData !== 'undefined';
+  useEffect(() => {
+    const status = (createManifestError as any)?.status;
```

Comment:
Suggestion: If you create an `HttpError` type, you'll be able to remove this `any` cast as well.